src/pages/ErrorPage/ErrorPage.tsx

```diff
@@ -0,0 +1,46 @@
+import { Link } from 'react-router-dom';
+import css from './ErrorPage.module.scss';
+
+import error from '~shared/error.png';
+
+type TErrorPageProps = {
+  statusCode: number;
+};
+
+const ErrorPage: React.FC<TErrorPageProps> = ({ statusCode }) => {
+  let errorMessage = '';
+  let errorDescription = '';
+
+  switch (statusCode) {
+    case 404:
+      errorMessage = '404';
+      errorDescription = 'We can’t find the page that you’re looking for.';
+      break;
+    case 505:
+      errorMessage = '505';
+      errorDescription = 'HTTP Version Not Supported';
+      break;
+    default:
+      errorMessage = '404';
+      errorDescription = 'We can’t find the page that you’re looking for.';
+      break;
+  }
```

Comment:
use an object with key - error/value - message instead of switch 
move it to separate file